### PR TITLE
[ts-sdk] Add utils export

### PR DESCRIPTION
**Pull request by hayes-mysten**
## Description 

Describe the changes or additions included in this PR.

## Test Plan 

How did you test the new or updated feature?

---
If your changes are not user-facing and not a breaking change, you can skip the following section. Otherwise, please indicate what changed, and then add to the Release Notes section as highlighted during the release process.

### Type of Change (Check all that apply)

- [ ] protocol change
- [ ] user-visible impact
- [ ] breaking change for a client SDKs
- [ ] breaking change for FNs (FN binary must upgrade)
- [ ] breaking change for validators or node operators (must upgrade binaries)
- [ ] breaking change for on-chain data layout
- [ ] necessitate either a data wipe or data migration

### CODE REVIEW

File: sdk/typescript/src/index.ts [hunk 1]
```diff
@@ -1,6 +1,8 @@
 // Copyright (c) Mysten Labs, Inc.
```
Comment (hayes-mysten) [inline sdk/typescript/src/index.ts:1]: I moved all the parts that have not been deprecated to the bottom of this file so it's easier to track whats left

File: sdk/typescript/src/utils/index.ts [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+	isValidTransactionDigest,
+	normalizeStructTag,
+	normalizeSuiAddress,
+	normalizeSuiObjectId,
+	parseStructTag,
+	SUI_ADDRESS_LENGTH,
+} from './sui-types.js';
+
+export { fromB64, toB64 } from '@mysten/bcs';
+export { is, assert } from 'superstruct';
```
Comment (Jordan-Mysten) [inline sdk/typescript/src/utils/index.ts:17]: I'm fine with these being here now but we really need to deprecate these exports, as we're planning on removing the concrete superstruct exports.

File: sdk/typescript/src/index.ts [hunk 3]
```diff
@@ -236,6 +258,8 @@ export {
 	Inputs,
 	/** @deprecated Import from `@mysten/sui.js/transactions` instead */
 	TransactionBlock,
+	/** @deprecated Import from `@mysten/sui.js/transactions` instead */
```
Comment (Jordan-Mysten) [inline sdk/typescript/src/index.ts:261]: Honestly this is exported by mistake, we really shouldn't even export this

Comment (Jordan-Mysten) [reply sdk/typescript/src/index.ts:261]: I'd be fine just marking as `deprecated` with no message because there's no real way to replace it

**Review by Jordan-Mysten — APPROVED** [review]

File: sdk/typescript/src/index.ts (hunk 3 above)
Comment (hayes-mysten) [reply sdk/typescript/src/index.ts:261]: this is used in a lot of places, but I don't really understand the transaction building stuff well enough to know why we can't replace it

Comment (Jordan-Mysten) [reply sdk/typescript/src/index.ts:261]: Outside of the SDK or inside of it?

Comment (hayes-mysten) [reply sdk/typescript/src/index.ts:261]: it's used once in the wallet, and a bunch in the kiosk sdk

Comment (Jordan-Mysten) [reply sdk/typescript/src/index.ts:261]: Hmm okay we can revisit later. I might be wrong and this might be useful

Comment (Jordan-Mysten) [reply sdk/typescript/src/index.ts:261]: WE probably just eventually want to make this a type-only export

Comment (hayes-mysten) [reply sdk/typescript/src/index.ts:261]: already made the new export type-only

**Pull request merged**